modules/ROOT/pages/cassandra/cassandra-get-started.adoc

```diff
@@ -1,12 +1,16 @@
-= Cassandra-Connector.adoc Connector - Get Started - Mule 4
+= Get Started - Mule 4
```

Comment:
The order of sections in this topic confuses me (I think this comes from the template)? And this doesn't feel like a getting started topic because there are no tasks.